static/app.template.html

```diff
@@ -123,6 +127,10 @@
   });
 });
 
+$(function() {
+     $("#datePicker").datepicker({ dateFormat: 'yy-mm-dd'}); 
```

Comment:
I use the data format that used to use.

I already use the "jquery-ui.css" (line 34) that embedded the datepicker css. Should I separete the datepicker pieces of the jquery-ui.css?